k8s/release/admin/secrets-csi-driver/secrets-csi-driver.yaml

```diff
@@ -12,7 +12,7 @@ spec:
   chart:
     repository: https://raw.githubusercontent.com/Azure/secrets-store-csi-driver-provider-azure/master/charts/
     name: csi-secrets-store-provider-azure
-    version: 0.0.20
+    version: 0.2.0
```

Comment:
don't you want to update this in just sandbox for now?

Reply:
added patches